docs/openrpc.json

```diff
@@ -0,0 +1,917 @@
+          "name": "data",
+          "schema": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      ],
+      "result": {
+        "name": "CreateTokenResponse",
+        "schema": {
+          "type": "object",
+          "oneOf": [
+            {
+              "description": "If push_tx parameter was false, wallet application returns transaction encoded in hexadecimal, ready to be pushed by any one."
+              "$ref": "#/components/schemas/TransactionHex"
+            },
+            {
+              "description": "If push_tx parameter was true, wallet application returns the transaction already registered in the blockchain."
+              "$ref": "#/components/schemas/Transaction"
+            }
+          ]
+        }
+      },
+      "errors": [
+        {
+          "$ref": "#/components/errors/ChangeAddressError"
+        },
+        {
+          "$ref": "#/components/errors/PromptRejectedError"
+        },
+        {
+          "$ref": "#/components/errors/CreateTokenError"
+        }
+      ]
+    },
+    {
+      "name": "htr_signOracleData",
+      "summary": "Requests oracle to sign data",
+      "description": "Provides an oracle user data to be signed and that a contract requires to work. dApp provides wallet application with data to be signed. Wallet application returns signed data.",
+      "paramStructure": "by-name",
+      "params": [
+        {
+          "$ref": "#/components/contentDescriptors/Network"
+        },
+        {
+          "name": "data",
+          "required": true,
+          "schema": {
+            "type": "string"
+          }
+        },
+        {
+          "name": "address",
+          "required": true,
+          "schema": {
+            "$ref": "#/components/schemas/Address"
+          }
+        }
+      ],
+      "result": {
+        "name": "SignOracleDataResponse",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "data": {
+              "type": "string"
+            },
+            "signature": {
+              "type": "string"
+            },
+            "oracle": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "errors": [
+        {
+          "$ref": "#/components/errors/DifferentNetworkError"
+        },
+        {
+          "$ref": "#/components/errors/PromptRejectedError"
+        }
+      ]
+    }
+  ],
+  "components": {
+    "contentDescriptors": {
+      "Network": {
+        "name": "network",
+        "required": true,
+        "schema": {
+          "$ref": "#/components/schemas/Network"
+        }
+      },
+      "Token": {
+        "name": "token",
+        "required": true,
+        "schema": {
+          "$ref": "#/components/schemas/Token"
+        }
+      }
+    },
+    "schemas": {
+      "Network": {
+        "enum": [
+          "testnet",
+          "mainnet"
+        ]
+      },
+      "Token": {
```

Comment:
Maybe this schema name can be `TransactionId` or `TransactionHash`, so it can reused. What do you think?

Reply:
OK: change made:
- I didn't change the name.
- I think it's important to keep the token schema.
- Instead of replacing it, I added a new "TransactionId" schema and now the Token schema only references it.